File: loops/notes.c
// Governor-Nephi Bakare, Loops Notes C
#include <stdio.h>


int main(void){
    //while loops
    int start = 0;
    while(start <5){
        printf("hello!\n");
        start++;
    }
    //for loops
    int q;
    for (q=0;q<5;q++){
        printf("%d\n", q);
    }

//8. How do you make lists in C?
    //Lists=Arrays in C
        //Arrays all need to be the same data type
    int grades[] = {88, 97, 100, 70, 72, 99, 61};
        //Step 1: set data type
        //Step 2: After naming, pyt brackets and writ the length of the list
        //Step 3: List surrounded by curly brackets {}
        //Step 4: Commas, between each item
//print a single item from a list
printf("CSP grade: %d\n", grades[1]);
//Change an item in the array
grades[1] = 95;
printf("CSP grade: %d\n", grades[1]);
//Size of list in bytes
int length = sizeof(grades);
printf("%d\n", length);
//length in list items
int length = sizeof(grades)/sizeof(grades[0]);
printf("The array is %d long\n", length);
//arary with strings
char movies[][55] = {"Cars", "Treasure Planet", "An American Tale", "Marley and Me", "The Avengers"};
printf("The movie is %s!\n", movies[1]);

int mlength = sizeof(movies)/sizeof(movies[0]);

//9. How do you make for loops in C?
int x;
//in parens *(starting point; ending point; what we're counting by)
for(x=0;x<=10;x+=2){
    printf("%d\n", x);
};
int m;
for(m=0;m<mlength;m++){
    
};
//10. How do you make while loops in C?
    return 0;
}

File: loops/notes.py
# Governor-Nephi Bakare, Loops Notes Python

#What is a loop? 
    #A section of code that repeats multiple times

#What are the two types of loops?
    #for loop
nums = [12,3,66,7,3,3,2]

for num in nums:
    print(num)

    #while loop
x = 0

while x < 10:
    print(x)
    x+= 1

#What is iteration
    #One instance of the loop
    #iteration is the amount of times it's looping through

#What are lists? 
    #A variable that holds multiple values
nums = [1,2,3,4,5,60]
siblings = ["Alex", "Genghis", "Xerxes"]
print(nums) #prints whole list is ugly
print(siblings[0]) #prints 1 item from the list *0 is the first one

siblings[0] = "Jake"
#siblings.pop(1)
#siblings.insert(2, "Deneeshaneesha")
#siblings.insert()
#print(siblings)
#How do you make lists in python? 
    #step 1: brackets *[]
    
    #step 2: add in items with correct data types *"" for words/letters, and none for numbers

    #step 3: add a comma in between each item 

#How do you make for loops in python? 
for sibling in siblings:
    print(sibling)

for x in range(1,5):
    print(x)

#How do you make while loops in python?
import random 
x = 1 #variable to keep count of iteration
goose = random.randint(1-20)

while x <=20:
    if x == goose:
       print ("Goose!")
       break #tells the loop to stop
    else:
        print("Duck!")
    x+= 1
    #print("Gigachad")